bin/hdfcoinc/pycbc_page_injtable

```diff
@@ -45,34 +73,42 @@ else:
             newsnr = pycbc.events.ranking.newsnr(snr, chisq)
             
             found_names += ["%s SNR" % ifo, "CHISQ", "NewSNR"]
-            found_cols += [snr, chisq, newsnr]
+            found_cols += [numpy.round(snr,2), numpy.round(chisq,2), numpy.round(newsnr,2)]
             found_formats += ['##.##', '##.##', '##.##']
-          
-dec_dist = numpy.maximum(inj['eff_dist_h'][:][idx], inj['eff_dist_l'][:][idx])
+
+eff_dist = {'eff_dist_l':'Eff Dist(L)','eff_dist_h':'Eff Dist(H)','eff_dist_v':'Eff Dist(V)',
+            'eff_dist_k':'Eff Dist(K)','eff_dist_i':'Eff Dist(I)'}
```

Comment:
again, try not to hard-code the IFOs. The involved IFOs will be in the attributes of the file

Reply:
This is just used to search for whether such an entry exists or not. I found even for two detector runs, there was eff distance entry for V1.

Reply:
So this is why we should use the ifos attribute of the injfind file - if V1 wasn't involved in the search, then V1 won't be in that attribute, and so you won't be looking for it in the injection file, even if it is there